src/ui/public/chrome/chrome.html

```diff
@@ -1,71 +1,48 @@
 <div class="content" chrome-context >
   <!-- TODO: These config dropdowns shouldn't be hard coded -->
-  <nav class="app-links-wrapper" ng-show="chrome.getVisible()">
```

Comment:
I like attributes on multiple lines because I tend to read the page top-to-bottom, and makes it a lot easier for me to scan. What are the advantages for keeping them on a single line?